src/interface/viewport.c

```diff
@@ -270,15 +270,56 @@ void sub_689174(sint16* x, sint16* y, sint16 *z)
 	*z = height;
 }
 
-// ax = viewport->x
-// bx = viewport->y;
-// cx = viewport->width;
-// dx = viewport->height;
+void sub_683326(int left, int top, int right, int bottom)
+{
+	RCT2_CALLPROC_X(0x00683359, left, top, right, bottom, 0, 0, 0);
+}
+
+// shifts pixels from the region in a direction. Used when a viewport moves;
+// consider putting in src/drawing/drawing.c or src/drawing/rect.c
+// 
+// 0x00683359
```

Comment:
can you use `/**/` for the comments at the start of functions. Also prefix the address of the function with `rct2: 0xBLAHBLAH`. In the past we have used this to tell how many functions we have implemented and this would mean this one is missed.